NitroxClient/GameLogic/Item.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using NitroxClient.Communication.Abstract;
 using NitroxClient.GameLogic.Helper;
 using NitroxClient.MonoBehaviours;
-using NitroxModel.DataStructures;
 using NitroxModel.DataStructures.Util;
 using NitroxModel.Logger;
 using NitroxModel.Packets;
-using NitroxModel_Subnautica.Helper;
+using NitroxModel_Subnautica.DataStructures;
```

Comment:
Instead, can we re-namespace unity/game surrogates?  This makes it harder to work with common structures such as NitroxId below.